lesson-07/task-07.js

```diff
@@ -0,0 +1,131 @@
+// =========================================
+// Task-01
+// =========================================
+
+const catigories = document.getElementsByTagName("h2");
```

Comment:
Помилочка в назві змінної :) 
```suggestion
const categories = document.getElementsByTagName("h2");
```